### DISCUSSION TITLE
账号被封了，理由是重复注册，麻烦站长帮忙看下。

### DISCUSSION THREAD

**Discussion by wdmzjxzl** [BurnHair/docs#117] [技术支持]
没有更改过github的账户名之类的。

之前我一直是正常使用的。不知道是不是因为每天签到白嫖过多了？账户余额有五十几了。但是我也一直没有怎么用。不知道怎么会被封禁，也没有做什么违规对话之类的。麻烦看下什么问题。

昨天晚上突然提示这个。

{ "error": { "message": "用户909已被封禁：重复注册 (request id: 202406231454572833610619HFJajZT)", "type": "one_api_error" } }

请问能不能解封一下或者怎么操作一下呢？

**Comment by BennyThink** [comment]
和签到之类的没关系。

7287和909 是同一人吧。
IP 91.243.81.37 139.162.85.167 等等。每次IP都一样

**Reply by wdmzjxzl** [reply]
是不是因为我在A电脑登录过一次，然后在B电脑上继续使用？误判了重复注册，因为最近我看因为这个原因被封封的用户也挺多的。

然后的话，能不能帮我解封一下呢？我是在公司电脑和家里电脑上行使用的。手机上也偶尔使用，真的不知道是怎么出发重复注册的，5555~
